internal/services/nss/nss.go

```diff
@@ -6,24 +6,25 @@ import (
 	"errors"
 
 	"github.com/ubuntu/authd"
-	"github.com/ubuntu/authd/internal/cache"
 	"github.com/ubuntu/authd/internal/log"
+	"github.com/ubuntu/authd/internal/newusers"
+	"github.com/ubuntu/authd/internal/newusers/cache"
```

Comment:
Yeah, I think we shouldn’t have that second import on "cache"

Reply:
I am still puzzled that we have to import cache though, unsure about the API is correct?

Reply:
Oh, it's because we are converting the `cache.NoDataFoundError` to `codes.NotFound` of gRPC.

Reply:
@didrocks, we forgot about this one

Reply:
I think the ideal way would be to remap the type (type alias) in `users/`?